### ISSUE TITLE
Can CIRI_DE_replicate be used for Differential expression analysis of DS_score

### ISSUE THREAD

**Issue by tjhwangxiong** [bioinfo-biols/CIRIquant#41]
Study without biological replicate, CIRI_DE generates DS_score.

In CIRI_DE_replicate output, only DE was generated.

I tried like this
`CIRI_DE_replicate --lib  library_info.csv \
            --bsj  circRNA_ratio.csv \
            --gene gene_count_matrix.csv \
            --out  circRNA_ds.tsv`
However, the FDR was all 1 for all circRNAs.

**Comment by Kevinzjy** [comment]
No, CIRI_DE can not deal with multiple samples, you can use statistical tests like t-test or rank sum to measure junction ratio changes.

**Comment by tjhwangxiong** [comment]
> No, CIRI_DE can not deal with multiple samples, you can use statistical tests like t-test or rank sum to measure junction ratio changes.

All my 10 samples (control: 5, case; 5) were sequenced using RNAse R pretreated RNA. I don't know whether the expression level of genes and transcripts is correct.

Moreover, 
`CIRI_DE_replicate --lib  library_info.csv \
            --bsj  circRNA_bsj.csv \
            --gene gene_count_matrix.csv \
            --out  circRNA_de.tsv`

Here is the sample list
`#sample.list
WT1 ./WT1/WT1.gtf C 1
WT2 ./WT2/WT2.gtf C 2
WT3 ./WT3/WT3.gtf C 3
WT4 ./WT4/WT4.gtf C 4
WT5 ./WT5/WT5.gtf C 5
AD1 ./FAD1/FAD1.gtf T 1
AD2 ./FAD2/FAD2.gtf T 2
AD3 ./FAD3/FAD3.gtf T 3
AD4 ./FAD4/FAD4.gtf T 4
AD5 ./FAD5/FAD5.gtf T 5`

I sorted the DE in circRNA_de.tsv, only 0, -1, 1 were found, is this normal?

Thanks.

**Comment by Kevinzjy** [comment]
The expression level of genes and transcripts could be quite different from normal poly(A) sequencing samples, and that's exactly the major limitation of circRNA analysis using RNase R treated libraries. But the circRNA differential expression results should be fine.

The final DE  column indicates whether the corresponding gene is significantly change (-1 = down-upregulated / 1 = up-regulated / 0 = no significant change).
